lms/djangoapps/teams/static/teams/js/spec/views/edit_team_members_spec.js

```diff
@@ -0,0 +1,137 @@
+define([
+    'jquery',
+    'underscore',
+    'backbone',
+    'common/js/spec_helpers/ajax_helpers',
+    'teams/js/views/edit_team_members',
+    'teams/js/models/team',
+    'teams/js/views/team_utils',
+    'teams/js/spec_helpers/team_spec_helpers'
+], function ($, _, Backbone, AjaxHelpers, TeamEditMembershipView, TeamModel, TeamUtils, TeamSpecHelpers) {
+    'use strict';
+
+    describe('CreateEditTeam', function() {
+        var editTeamID = 'av',
+            DEFAULT_MEMBERSHIP = [
+                {
+                    'user': {
+                        'username': 'frodo',
+                        'profile_image': {
+                            'has_image': true,
+                            'image_url_medium': '/frodo-image-url'
+                        },
+                        last_activity_at: "2015-08-21T18:53:01.145Z",
+                        date_joined: "2014-01-01T18:53:01.145Z"
+                    }
+                }
+            ],
+            deleteTeamMemember = function (view, confirm) {
+                view.$('.action-remove-member').click();
+                // Confirm delete dialog
+                if (confirm) {
+                    $('.action-primary').click();
+                }
+                else {
+                    $('.action-secondary').click();
+                }
+            },
+            verifyTeamMembersView = function (view) {
+                expect(view.$('.team-member').length).toEqual(1);
+                expect(view.$('.member-profile').attr('href')).toEqual('/u/frodo');
+                expect(view.$('img.image-url').attr('src')).toEqual('/frodo-image-url');
+                expect(view.$('.member-info-container .primary').text()).toBe('frodo');
+                expect(view.$el.find('#last-active').text()).toBe("2015-08-21T18:53:01.145Z");
+                expect(view.$el.find('#date-joined').text()).toBe("2014-01-01T18:53:01.145Z");
```

Comment:
These lines are causing the current js test failures, I'll have them fixed up shortly